thorlcr/activities/msort/thsortu.cpp

```diff
@@ -474,6 +468,13 @@ class CJoinHelper : implements IJoinHelper, public CSimpleInterface
         outputmetaL = _outputmeta;
         if (TAKdenormalize == kind || TAKhashdenormalize == kind || TAKlookupdenormalize == kind || TAKsmartdenormalize == kind)
             denormTmp.setAllocator(allocator).ensureRow();
+        denormCount = 0;
+        joinCounter = 0;
+        outSz = 0;
+        rightidx = 0;
+        leftmatched = false;
+        lhsProgressCount = 0;
+        rhsProgressCount = 0;
```

Comment:
ok, why not all the same ones as before ?

Reply:
they should be, which ones look different?

Reply:
Maybe I missed it, is this one there ?
```
        limitedCompareR = NULL;
```

Reply:
ok, maybe it is unused and removed ?

Reply:
ah that one - yes removed - it was not used.